### DISCUSSION TITLE
Title for the Game?

### DISCUSSION THREAD

**Discussion by jeffaroundtheworld** [jeffaroundtheworld/2024gamecollab#96] [General]
The start screen is noticeably lacking a title for the game... we should add one - but first we need a title - and 2024collab game is a terrible one.

Game title ideas?

**Comment by JackV13** [comment]
Haunted Halls

**Comment by ajleach9689** [comment]
Shadows of St. Agnes

**Comment by kodriq** [comment]
Balloon tower defense 4

**Reply by jeffaroundtheworld** [reply]
Rhys says this is already taken

**Comment by kodriq** [comment]
Impelled Instinct
^The games technically about being forced down to your survival instincts
